autosklearn/estimators.py

```diff
@@ -396,6 +403,9 @@ def __init__(
         self.initial_configurations_via_metalearning = (
             initial_configurations_via_metalearning
         )
+
+        # Need to resolve the ensemble class here so we can act on it below.
+        ensemble_class = self.resolve_ensemble_class(ensemble_class, metric)
```

Comment:
I moved this method into init since it's fairly short and it exposes less methods to the user that they probably don't need to know about.

Reply:
I'm a bit confused about your comment here. This was in __init__ before your changes and still is in __init__ after your changes.

Reply:
This was an outdated comment and reverted with one of the commits #[18b6db4](https://github.com/automl/auto-sklearn/pull/1523/commits/18b6db487698cc21adea9674727117638d5ce41d).

I think the goal was just to remove the method but it didn't make sense to do so in the end.